examples/cfd/04_burgers.ipynb

```diff
@@ -201,53 +195,25 @@
   {
    "cell_type": "code",
    "execution_count": 5,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [
     {
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "DSE: extract_time_invariants [flops: 56, elapsed: 0.01] >>\n",
-      "     eliminate_inter_stencil_redundancies [flops: 56, elapsed: 0.00] >>\n",
-      "     eliminate_intra_stencil_redundancies [flops: 56, elapsed: 0.02] >>\n",
-      "     factorize [flops: 56, elapsed: 0.00] >>\n",
-      "     finalize [flops: 68, elapsed: 0.00]\n",
-      "     [Total elapsed: 0.03 s]\n",
-      "DSE: extract_time_invariants [flops: 0, elapsed: 0.00] >>\n",
-      "     eliminate_inter_stencil_redundancies [flops: 0, elapsed: 0.00] >>\n",
-      "     eliminate_intra_stencil_redundancies [flops: 0, elapsed: 0.00] >>\n",
-      "     factorize [flops: 0, elapsed: 0.00] >>\n",
-      "     finalize [flops: 0, elapsed: 0.00]\n",
-      "     [Total elapsed: 0.01 s]\n",
-      "DSE: extract_time_invariants [flops: 0, elapsed: 0.00] >>\n",
-      "     eliminate_inter_stencil_redundancies [flops: 0, elapsed: 0.00] >>\n",
-      "     eliminate_intra_stencil_redundancies [flops: 0, elapsed: 0.00] >>\n",
-      "     factorize [flops: 0, elapsed: 0.00] >>\n",
-      "     finalize [flops: 0, elapsed: 0.00]\n",
-      "     [Total elapsed: 0.01 s]\n",
-      "DLE: analyze [elapsed: 0.01] >>\n",
-      "     avoid_denormals [elapsed: 0.00] >>\n",
-      "     loop_fission [elapsed: 0.00] >>\n",
-      "     loop_blocking [elapsed: 0.02] >>\n",
-      "     simdize [elapsed: 0.12] >>\n",
-      "     create_elemental_functions [elapsed: 0.00] >>\n",
-      "     minimize_remainders [elapsed: 0.00]\n",
-      "     [Total elapsed: 0.16 s]\n",
-      "CustomCompiler: compiled /tmp/devito-1000/1645e03592558a800a6ea8efe72867472be621c5.c [0.25 s]\n",
+      "CustomCompiler: compiled /var/folders/qr/4hc1nw7d3hj55jbd1z7kkrvh0000gn/T/devito-y2kr1ngp/9beba14a9cc90a026c15251528633e01066ebb49.c [0.64 s]\n",
       "=========================================================================================\n",
-      "Section section_1<120,39> with OI=0.00 computed in 0.000 s [Perf: 0.00 GFlops/s]\n",
-      "Section section_2<120,39> with OI=0.00 computed in 0.000 s [Perf: 0.00 GFlops/s]\n",
-      "Section main<120,39,39> with OI=3.81 computed in 0.002 s [Perf: 6.60 GFlops/s]\n",
+      "Section section_1<121,41> with OI=0.00 computed in 0.000 s [0.00 GFlops/s]\n",
+      "Section section_2<121,41> with OI=0.00 computed in 0.000 s [0.00 GFlops/s]\n",
+      "Section main<121,41,41> with OI=4.25 computed in 0.005 s [2.96 GFlops/s, 0.04 GPts/s]\n",
```

Comment:
Another place where iteration spaces and OI have changed.

Reply:
same as before